docs/source/core_services/service_agents/central_management/VOLTTRON-Central.rst

```diff
@@ -46,7 +46,7 @@ Verify that the instance is serving properly by pointing your web browser to
 
 ::
 
-    http://127.0.0.1:8080/discovery/
+    http://127.0.0.1:8080/vc/discovery/
```

Comment:
this one should stay as /discovery/ as it's not a vc specific command...this one is available to any caller.

Reply:
done